Makefile

```diff
@@ -320,7 +320,7 @@ ctl:
 # A special target for testing only "coprocessor::dag::expr"
 # per https://github.com/tikv/tikv/pull/3280
 expression: format clippy
-	RUST_BACKTRACE=1 cargo test --features "${ENABLE_FEATURES}" --no-default-features --package tidb_query "expr" -- --nocapture
+	RUST_BACKTRACE=1 cargo test --features "${ENABLE_FEATURES}" --no-default-features --package tidb_query_datatype "expr" -- --nocapture
```

Comment:
This section is outdated. I think we can simply remove them because I believe no one actually uses it for a long time...